scripts/upgrade

```diff
@@ -48,7 +48,7 @@ cli_port=$(ynh_app_setting_get $app cli_port)
 #=================================================
 
 # To be sure that the migration is sucessfull we check that the old synapse version is compatible with the synapse_port_db script.
-if [[ -z $synapse_old_version ]] && [[ $(dpkg -l | grep -c -E "ii.*matrix-synapse.*0.25") != 1 ]] && [[ $(dpkg -l | grep -c -E "ii.*matrix-synapse.*0.26") != 1 ]]
+if [[ -z $synapse_old_version ]] && [[ $(dpkg -l | grep -c -E "ii.*matrix-synapse.*0.27") != 1 ]] && [[ $(dpkg -l | grep -c -E "ii.*matrix-synapse.*0.26") != 1 ]]
```

Comment:
Why do you need to modify this line ?

Reply:
I implemented this strategy to be sure that we upgrade a old instance how as a database compatible with the new instance. [You can see how many issue there are about that.](https://github.com/matrix-org/synapse/issues?utf8=%E2%9C%93&q=is%3Aissue+synapse_port_db)

So by this line we check that we are not in a case of an old instance" (how the synapse source is managed by apt) upgraded to a version how has a broken port_db script.

Reply:
Hum, that doesn't really answer my question.

I understood that the idea is to check if the previous version will allow an upgrade without crashing.
But, why do you need to modify the version number here ?

Actually, I think the better thing would probably to change this line to something clearer. Or to explain each part of the line maybe.

Reply:
Because, synapse on the apt repose has been also updated. So we need to make the user to be able to upgrade to the new package (if we have checked before that this version is supported by the script).

> Actually, I think the better thing would probably to change this line to something clearer. Or to explain each part of the line maybe.

Ok, but we just allow the upgrade for the two last version.

Reply:
Sorry for the delay...

Your explanation is quite clear. If you can just add this as an commentary in your code I think it would be clearer for all :)